doc_source/use-cases-distribution.md

```diff
@@ -1,7 +1,7 @@
 # Use Case: Distribution<a name="use-cases-distribution"></a>
 
-You can use AWS Elemental MediaConnect to distribute your content to different geographical locations\. For example, suppose that your on\-premises contribution encoder is located in Portland, Oregon and your receivers are located around the world\. \(A receiver is any entity that will receive content from your flow\. This could be an encoder in the cloud, an on\-premises encoder at your recipient facility, or another AWS Elemental MediaConnect flow\.\) You set up your initial AWS Elemental MediaConnect flow in the us\-west\-1 Region, which is the closest physical AWS Region to your encoder\. After your content is in the AWS Cloud, you send it to other MediaConnect flows located in Regions that are closer to your receivers\.
+You can use AWS Elemental MediaConnect to distribute content to different geographic regions\. 
 
-This following illustration shows an on\-premises contribution encoder located in Portland, Oregon that uploads content to AWS Elemental MediaConnect in the AWS Cloud\. The flow has three outputs that send content to others flows in different AWS Regions\. These secondary flows are closer to the receivers, which are located in various cities around the world\.
+Assume that you have an on\-premises contribution encoder in Portland, Oregon. You can set up uploads through an AWS Elemental MediaConnect cloud system in the us\-west\-1 Region, closest to Portland. You can then configure that AWS Elemental MediaConnect instance to send your content to MediaConnect systems in other regions, as depicted in the following diagram.
```

Comment:
No need to start with "The following illustration", as the illustration is there on the page.

When discussing the use case, I think it's helpful to take the PoV of the reader, to help that reader think: "Oh, that's what I can do!"